deploymentscripts/lib/bash/ubuntu-prep.sh

```diff
@@ -14,8 +14,8 @@ echo 'eval "$(rbenv init -)"' >> ~/.bashrc
 export PATH="$HOME/.rbenv/bin:$PATH"
 eval "$(rbenv init -)"
 git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
-RUBY_CONFIGURE_OPTS=--disable-install-doc rbenv install 2.6.5
-rbenv global 2.6.5
+RUBY_CONFIGURE_OPTS=--disable-install-doc rbenv install 2.7.2
+rbenv global 2.7.2
 gem install bundler -v 1.16.1 --no-rdoc --no-ri
```

Comment:
```
-gem install bundler -v 1.16.1 --no-rdoc --no-ri
+gem install bundler -v 1.17.2
```

We need to make this change as well.  I'm not sure if the version bump is critical, but it looks like in 2.7.2 the --no-rdoc and --no-ri flags cause fatal errors instead of just being ignored.

Reply:
Good catch! According to the [Ruby 2.6.0 release notes](https://www.ruby-lang.org/en/news/2018/12/25/ruby-2-6-0-released/) we can use `--no-document` instead.
> Merge RubyGems 3.0.1. The --ri and --rdoc options have been removed. Please use the --document and --no-document options instead.